src/clarity/wizard/wizard.spec.ts

```diff
@@ -466,6 +480,15 @@ describe("Wizard", () => {
             expect(tab3).toBeDefined();
             expect(tab4).toBeDefined();
         });
+
+        it("calls the user-defined onCancel handler when the Cancel button is clicked", () => {
+            fixture.detectChanges();
+            doCancel(compiled);
+
+            expect(fixture.componentInstance.hasBeenCanceled).toBe(true);
+            spyOn(instance, "_close");
```

Comment:
Please avoid testing the internal implementation of the component. I think making sure the output is emitted is enough for this test.